src/cmdc_tools/datasets/official/AK/__init__.py

```diff
@@ -32,7 +32,7 @@
 
 def _find_col_by_prefix(df, col_prefix):
     cols = [x for x in list(df) if x.lower().startswith(col_prefix.lower())]
-    assert len(cols) == 1
```

Comment:
Hmm are we confident this is correct? What are the two columns that are found?

Reply:
Yes I looked at it. They are `Community` and `Community\nCases`

Reply:
What is the difference between the two in terms of data? I see that later in the PR you started using the names with `\n` in them.